sql/updates/world/3.3.5/2099_25_12_00_world.sql

```diff
@@ -0,0 +1,45 @@
+DELETE FROM `conditions` WHERE `SourceTypeOrReferenceId`=13 AND `SourceEntry`=64799; -- Lightning Tether
+INSERT INTO `conditions` (`SourceTypeOrReferenceId`, `SourceGroup`, `SourceEntry`, `SourceId`, `ElseGroup`, `ConditionTypeOrReference`, `ConditionTarget`, `ConditionValue1`, `ConditionValue2`, `ConditionValue3`, `NegativeCondition`, `ErrorType`, `ErrorTextId`, `ScriptName`, `Comment`) VALUES
+(13,1,64799,0,0,31,0,3,33293,0,0,0,0,'','Effect_0 hits XT002');
+
+DELETE FROM `spell_script_names` WHERE `ScriptName` IN (
+'spell_xt002_exposed_heart',
+'spell_xt002_energy_orb');
+INSERT INTO `spell_script_names` (`spell_id`, `ScriptName`) VALUES
+(63849,'spell_xt002_exposed_heart'),
+(62826,'spell_xt002_energy_orb');
+
+DELETE FROM `spell_proc` WHERE `SpellId`=63849; -- Disable spell proc to effect_1
+INSERT INTO `spell_proc` (`SpellId`, `SchoolMask`, `SpellFamilyName`, `SpellFamilyMask0`, `SpellFamilyMask1`, `SpellFamilyMask2`, `ProcFlags`, `SpellTypeMask`, `SpellPhaseMask`, `HitMask`, `AttributesMask`, `ProcsPerMinute`, `Chance`, `Cooldown`, `Charges`) VALUES
+(63849,0,0,0,0,0,0,0,0,0,32,0,0,0,0);
+
+UPDATE `creature` SET `MovementType`=0, `Spawndist`=0 WHERE `id`=33337; -- XT-Toy Pile
+
+UPDATE `creature_template` SET `speed_walk`=1.2, `speed_run`=0.428571, `ScriptName`='npc_scrapbot' WHERE `entry`=33343;
+UPDATE `creature_template` SET `speed_walk`=1.2, `speed_run`=0.428571 WHERE `entry`=33887; -- Scrapbot
+UPDATE `creature_template` SET `speed_walk`=1.2, `speed_run`=0.428571, `modelid2`=0, `modelid3`=0 WHERE `entry` IN (33346,33886); -- Boombot
+UPDATE `creature_template` SET `BaseAttackTime`=2000 WHERE `entry` IN (33293,33885); -- XT-002 Deconstructor (hi ariel, I stole from you xD)
+UPDATE `creature_template` SET `mechanic_immune_mask`=617299839 WHERE `entry` IN (33329,33885); -- Heart mechanical immunities xD
+
+DELETE FROM `waypoint_scripts` WHERE `id` IN(2,5);
```

Comment:
not is better add guid in DELETE ??

Reply:
If you refer to `guid` being `PRIMARY KEY`, it's a fair point.
Then again, if there is more than one of `id`, it will remove duplicate `id`s.

Reply:
```diff
-DELETE FROM `waypoint_scripts` WHERE `id` IN(2,5);
+DELETE FROM `waypoint_scripts` WHERE `id`=2 AND `guid` IN (1360540,1360541);
+DELETE FROM `waypoint_scripts` WHERE `id`=5 AND `guid` IN (1360542,1360543);
```

Reply:
Not need, waypoint_scripts normally is used by 1 creature only.
And still, has no others waypoints scripts in id 2 or 5.

Reply:
Well, it depends on the intended functionality of the `DELETE FROM` query.
As of now (4fe960b9e87174233d1fec9734bf34f38ad2cc5f), there are no existing `waypoint_scripts` using either `id` IN (2, 5) or `guid` IN (1360540,1360541,1360542,1360543).
```sql
SELECT * FROM `waypoint_scripts` WHERE `id` IN (2,5) OR `guid` IN (1360540,1360541,1360542,1360543);
```
- 0 row(s)

Because of this, our discussion here is mostly academic.

Reply:
Yep, I simply indicate the option that seems most appropriate to me in a generic way 😄

Reply:
The existing parameters in the query is enough.